integration/testnet/network.go

```diff
@@ -793,7 +819,7 @@ func setupKeys(networkConf NetworkConfig) ([]ContainerConfig, error) {
 		// define the node's name <role>_<n> and address <name>:<port>
 		name := fmt.Sprintf("%s_%d", conf.Role.String(), roleCounter[conf.Role]+1)
 
-		addr := fmt.Sprintf("%s:%d", name, 2137)
+		addr := fmt.Sprintf("%s:%d", name, DefaultStakedFlowPort)
```

Comment:
Depending on whether the node is staked or not, we should use `DefaultUnstakedFlowPort` instead.